browser/index.js

```diff
@@ -66,19 +78,13 @@ module.exports = (opts, userPlugins = []) => {
   const bugsnag = new Client({ name, version, url }, finalSchema)
 
   // set transport based on browser capability (IE 8+9 have an XDomainRequest object)
-  bugsnag.transport(window.XDomainRequest ? transports.XDomainRequest : transports.XMLHttpRequest)
-
-  try {
-    // configure with user supplied options
-    // errors can be thrown here that prevent the lib from being in a useable state
-    bugsnag.configure(opts)
-  } catch (e) {
-    bugsnag._logger.warn(e)
-    if (e.errors) map(e.errors, bugsnag._logger.warn)
-    // rethrow. if there was an error with configuration
-    // the library is not going to work
-    throw e
-  }
+  bugsnag.transport(window.XDomainRequest ? tXDomainRequest : tXMLHttpRequest)
```

Comment:
Is this no longer capable of throwing an error?

Reply:
It throws errors still, and we stop swallowing them. This commit detail should explain things in more detail https://github.com/bugsnag/bugsnag-js/pull/341/commits/5778437e540cc55bfe48ab38f88c2e14f8638bb6.